aws_lambda.py

```diff
@@ -4,8 +4,20 @@
 import os
 import sys
 
+import sentry_sdk
+from sentry_sdk.integrations.aws_lambda import AwsLambdaIntegration
+from decouple import config
 
-def help(**kwargs):
+SENTRY_DSN = config("SENTRY_DSN", default=None)
+
+if SENTRY_DSN:
+    # Note! If you don't do `sentry_sdk.init(DSN)` it will still work
+    # to do things like calling `sentry_sdk.capture_exception(exception)`
+    # It just means it's a noop.
+    sentry_sdk.init(SENTRY_DSN, integrations=[AwsLambdaIntegration()])
```

Comment:
Incidentally, and I tested this, even if you're just running a regular virtualenv, this integration still works in terms of actually sending real exceptions to Sentry.

Reply:
:+1: